package.json

```diff
@@ -1,43 +1,46 @@
 {
   "name": "framework",
-  "version": "0.0.1",
+  "keywords": [],
+  "version": "1.0.0",
   "description": "This is the SumoCoders Symfony Framework.",
   "repository": {
     "type": "git",
     "url": "git://github.com/sumocoders/Framework.git"
   },
-  "author": "Tijs Verkoyen",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/sumocoders/Framework/issues"
   },
   "homepage": "https://github.com/sumocoders/Framework",
+  "author": "Tijs Verkoyen",
+  "contributors": [
+    "Mathias Helin",
+    "Katrien Vanhaute"
+  ],
+  "license": "MIT",
+  "dependencies": {},
   "devDependencies": {
-    "bower": "^1.7.9",
+    "bootstrap-sass": "^3.3.7",
+    "font-awesome": "^4.7.0",
```

Comment:
word dit mee gedeployed? zo niet dan zal dit niet werken want de fonts zitten in de node module map